org-eclipse-che-jdt-ls-extension/jdt.ls.extension.test/projects/maven/usages/pom.xml

```diff
@@ -20,4 +20,26 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.6.1</version>
+        <configuration>
+          <compilerId>eclipse</compilerId>
+          <source>1.8</source>
+          <target>1.8</target>
+          <forceJavacCompilerUse>true</forceJavacCompilerUse>
```

Comment:
forecJavacCompilerUse and dependency on the eclipse compiler? Is that correct?

Reply:
Actually i don't know :) Will try without them.

Reply:
fixed